R/SS_tune_comps.R

```diff
@@ -193,8 +185,8 @@
 SS_tune_comps <- function(replist = NULL, fleets = "all",
                           option = c("Francis", "MI", "none", "DM"),
                           digits = 6, write = TRUE, niters_tuning = 0,
-                          init_run = FALSE, dir = getwd(), model = "ss",
```

Comment:
I wonder if `model` should be formally deprecated using lifecycle::deprecated, instead of just changing the name to exe?

Reply:
Done, thanks for the reminder.